src/FixtureBuilder/Denormalizer/Fixture/SimpleFixtureBagDenormalizer.php

```diff
@@ -69,22 +75,30 @@ public function denormalize(array $data): FixtureBag
                     )
                 );
             }
+            try {
+                foreach ($rawFixtureSet as $reference => $specs) {
+                    if (null === $specs) {
+                        $specs = [];
+                    }
 
-            foreach ($rawFixtureSet as $reference => $specs) {
-                if (null === $specs) {
-                    $specs = [];
+                    $fixtures = $this->fixtureDenormalizer->denormalize(
+                        $fixtures,
+                        $fqcn,
+                        $reference,
+                        $specs ?? [],
+                        $flags
+                    );
+                }
+            } catch (UnexpectedValueException $e) {
```

Comment:
From where is this exception thrown?

Reply:
The exception is triggered inside the  `get` call in https://github.com/nelmio/alice/blob/19bec34576564cb54921e096bf0e5eba2bee027e/src/FixtureBuilder/Denormalizer/Fixture/Chainable/ReferenceRangeNameDenormalizer.php#L130 and is the core of this PR. 

When `get` is called, the necessary fixture to reference has not yet been loaded, with the `while` loop introduced in this PR, we retry once more at the end of the loading process.